.github/workflows/ci.yaml

```diff
@@ -37,3 +37,29 @@ jobs:
         type=edge,suffix=-{{date 'X'}}
         type=match,pattern=multipathd-exporter-v(.*),group=1
         type=match,pattern=multipathd-exporter-v(.*),group=1,value=latest
+
+  chart-testing-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: azure/setup-helm@v4.2.0
+      - uses: helm/chart-testing-action@v2.7.0
+      - run: ct lint --charts charts/prometheus-multipathd-exporter
```

Comment:
Ok, sorry for this, I just learned that there is a `ct lint-and-install` command, which will lint, install and test.